sympy/core/relational.py

```diff
@@ -543,6 +543,49 @@ def _eval_simplify(self, ratio, measure, rational, inverse):
                 pass
         return e.canonical
 
+    def subs(self, *args, **kwargs):  # should mirror sympy.core.basic.subs
+        r"""Return a new equation with subs applied to both sides.
```

Comment:
```suggestion
        r"""Allows substitution of several equalities into an equation with the same variables, assuming that lhs is substituted by rhs.
```

I think something like this or similar to this would be a better description of the function rather than what it returns.

Reply:
Also, it might be a good idea to include a warning / note that this function cannot be used for substituting inequalities or equalities into expressions or inequalities.